<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateCustomer</class>
 <widget class="QDialog" name="UpdateCustomer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter name of customer you would like to update:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Customer Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_customerName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter new values for the fields to be updated:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Select">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_message">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>501</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sreet Address:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>City</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>State</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Zipcode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Key Customer (1-0)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Interest</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_StreetAddress">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_City">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_State">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_zipCode">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_keyCust">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Interest">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
